### ISSUE TITLE
spelling of Prandtl in Fluid.Dissipation

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1135]
**Reported by anonymous on 28 May 2013 09:11 UTC**
In Fluid.Dissipation, the following line appears several times:

```
output SI.PrandtlNumber Pr "Prandl number" annotation (Dialog(group="Output"));
```

it should instead be

```
output SI.PrandtlNumber Pr "Prandtl number" annotation (Dialog(group="Output"));
```

As it is a comment only, it is not urgent at all

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1135

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 28 May 2013 09:16 UTC**
Since it's an easy one to fix...

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 28 May 2013 09:21 UTC**
Fixed in 2c86057c2dfe1b417a9a7ac62b6c9e4e34235447

**Comment by modelica-trac-importer** [comment]
**Changelog modified by dietmarw on 28 May 2013 09:21 UTC**
Fixed a typo in the name "Prandtl".
